### DISCUSSION TITLE
Good morning

### DISCUSSION THREAD

**Discussion by gysnst2** [Snapmaker/Luban#982] [General]
Good morning snappers

**Comment by parachvte** [comment]
I've updated your feature request open earlier today. It's not well formatted.

https://github.com/Snapmaker/Luban/issues/981

**Reply by gysnst2** [reply]
I saw that I'm trying to figure out the format you guys are looking for.  

To add to the feature we are looking to have the lights power on and stay on on power up.  It is 50/50 some want them off on power up.  But we all agree we want their to be a setting that can be saved so whatever option we want becomes the default
